src/core/labeling/qgspallabeling.h

```diff
@@ -321,6 +321,7 @@ class CORE_EXPORT QgsPalLayerSettings
       // (data defined only)
       PositionX = 9, //!< X-coordinate data defined label position
       PositionY = 10, //!< Y-coordinate data defined label position
+      PositionPoint = 79, //!< Point-coordinate data defined label position
```

Comment:
```suggestion
      PositionPoint = 114, //!< Point-coordinate data defined label position
```

I vaguely recall that an entry was removed from this enum in 3.0, which would account for the gap. If that's so then we should avoid reusing the possibly conflicting value